spec/serializers/topic_view_serializer_spec.rb

```diff
@@ -64,4 +66,44 @@
       end
     end
   end
+
+  let(:user) { Fabricate(:user) }
+  let(:moderator) { Fabricate(:moderator) }
+  let(:tag) { Fabricate(:tag) }
+  let(:pm) do
+    Fabricate(:private_message_topic, tags: [tag], topic_allowed_users: [
+      Fabricate.build(:topic_allowed_user, user: moderator),
+      Fabricate.build(:topic_allowed_user, user: user)
+    ])
+  end
+
+  describe 'when tags added to private message topics' do
+    before do
+      SiteSetting.tagging_enabled = true
+      SiteSetting.allow_staff_to_tag_pms = true
+    end
+
+    it "should not include the tag for normal users" do
+      json = serialize_topic(pm, user)
+      expect(json[:tags]).to eq(nil)
+    end
+
+    it "should include the tag for staff users" do
+      json = serialize_topic(pm, moderator)
+      expect(json[:tags]).to eq([tag.name])
+
+      json = serialize_topic(pm, Fabricate(:admin))
```

Comment:
`Fabricate(:admin)` can be moved into `let(:admin)` just like how it is done for `moderator`